packages/server/src/app/db/comments/comments.service.ts

```diff
@@ -34,6 +37,16 @@ export class CommentsService {
 
         let doc = await newComment.save();
         await this.blogsService.addComment(blogId);
+
+        // Send notification to blog author
+        await this.notificationsService.queueNotification({
+            sourceId: doc._id,
+            sourceKind: NotificationSourceKind.Comment,
+            sourceParentId: blogId,
+            sourceParentKind: NotificationSourceKind.Blog,
+            title: `${user.username} posted a new comment on your blog!` // todo: add in blog title here?
```

Comment:
This is one of the areas where the current data model kind of breaks down. It feels silly to stop, and go make a DB request for both:
1) The commenter's user name
2) The blog's title
...so I really feel like I need to spend some more time thinking about what a client actually needs to construct a useful notification.